### ISSUE TITLE
[Fortinet - ResponseOnIP Playbook ] Deployment Fails 

### ISSUE THREAD

**Issue by melv33n** [Azure/Azure-Sentinel#8985]
**Describe the bug**
When trying to use SystemAssigned Identity on [Fortinet - ResponseOnIP](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/Fortinet-FortiGate/Playbooks/Fortinet_ResponseOnIP) Playbook fails on deployment. Every Pre-requisite is satisfied:

[Custom connector](https://github.com/Azure/Azure-Sentinel/blob/master/Playbooks/Fortinet-FortiGate/CustomConnector)
[Function App](https://github.com/Azure/Azure-Sentinel/blob/master/Playbooks/Fortinet-FortiGate/(./FunctionApp))

Deployment prompts:
![imagen](https://github.com/Azure/Azure-Sentinel/assets/18287599/6d0d7a4c-8ef7-4cfa-ae5e-13ff27fac468)

`{
      "code": "MissingIdentityIds",
      "message": "The identity ids must not be null or empty for 'UserAssigned' identity type."
    }`

It seems that the template restores the values and doesn't allows you to use SystemAssigned instead of UserAssigned identities. Other playbooks allow you to do it. 

**To Reproduce**
1. Follow the deployment pre-requisites mentioned in [fortinet folder](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/Fortinet-FortiGate)
2. Select the template
![imagen](https://github.com/Azure/Azure-Sentinel/assets/18287599/c5ec1392-ee86-43a3-96e7-78f00954e515)
3. Set Name and procced to parameters tab
4. Insert SystemManaged identity name
![imagen](https://github.com/Azure/Azure-Sentinel/assets/18287599/cf69bbb8-3bf5-41d0-9710-0acdfb330af7)
5. See the deployment Error
![imagen](https://github.com/Azure/Azure-Sentinel/assets/18287599/cdcdb6b5-a309-4bc1-a429-a78a5063c037)

**Comment by v-rbajaj** [comment]
Hi @melv33n, we are trying to reproduce this issue on our end and will get back to you once there is an update

**Comment by v-rbajaj** [comment]
Hi @melv33n, we are still trying to reproduce this issue on our end and will get back to you once there is an update

**Comment by v-rbajaj** [comment]
Hi @melv33n, we have reached out to concerned team from our end and will get back to you by 17 Oct 2023

**Comment by v-rbajaj** [comment]
Hi @melv33n, we have reached out to concerned team from our end and will get back to you by 23 Oct 2023

**Comment by v-rbajaj** [comment]
Hi @melv33n, sorry for delay, we have reached out to concerned team from our end and will get back to you by 27 Oct 2023

**Comment by v-rbajaj** [comment]
Hi @melv33n, sorry for delay, we have reached out to concerned team from our end and will get back to you by 31 Oct 2023

**Comment by v-rbajaj** [comment]
Hi @melv33n, sorry for delay, we have reached out to concerned team from our end and will get back to you by 03 Nov 2023

**Comment by manishkumar1991** [comment]
@melv33n , 

We've created FortiGate playbooks specifically tailored to operate based on user identity. This is explicitly stated within the playbook's code. 

![image](https://github.com/Azure/Azure-Sentinel/assets/97503740/03c4f028-3194-4bb6-8c7c-6d68f5d031dd)


Considering your note about system-assigned managed identity functioning with other playbooks, we attempted a similar approach with a different Fortinet FortiGate playbook, but encountered an error. 

![image](https://github.com/Azure/Azure-Sentinel/assets/97503740/2385ee5e-9f6e-4333-b36e-74fb0fc3331d)

Your feedback will be taken into account and added to our backlog for future consideration, although we cannot currently provide an estimated time of resolution.
